readme.md

```diff
@@ -146,7 +169,7 @@ You can bind your own shortcuts in normal mode with the `:bind` command. For exa
     -   addons.mozilla.org is now supported so long as you run `fixamo` first.
 -   Tridactyl now supports changing the Firefox GUI if you have the native messenger installed via `guiset`. There's quite a few options available, but `guiset gui none` is probably what you want, perhaps followed up with `guiset tabs always`. See `:help guiset` for a list of all possible options.
 
-## Frequently asked questions (FAQ)
+## Frequently Asked Questions
```

Comment:
I really don't like Title Case (Where Every Letter Is Captilised). Do we use it elsewhere? If we do, I'd rather we changed those instances.

I think (FAQ) in brackets was added more for people searching the page than it was for people reading the title, if you see what I mean. I agree that it's ugly but I think it serves a purpose.

So I'd leave this line alone. Sorry for the bikeshedding : )